td.vue/package.json

```diff
@@ -58,6 +60,8 @@
     "browserstack-cypress-cli": "^1.8.1",
     "chromedriver": "^90.0.0",
     "codecov": "^3.8.2",
+    "electron": "^13.0.0",
```

Comment:
The latest  version of electron is 17.0.0 - I have tried it with this version and it seems to work OK

Reply:
Done!